nativelib/src/main/scala/scala/scalanative/native/posix/posixErrno.scala

```diff
@@ -0,0 +1,8 @@
+package scala.scalanative.native.posix
+
+import scala.scalanative.native.{CInt, name, extern}
+
+object posixErrno {
```

Comment:
Lets name it just `errno`, it should be clear that this is `posix`'s version of `errno`.